spec/order_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'order'
+
+describe Order do
+
+  let(:menu) do
+
+    double :menu, view: {pepperoni:  10.0,
+                         margharita:  8.0}
+
+  end
+
+  subject {Order.new(menu)}
+
+  it {expect(subject).to respond_to(:add).with(2).argument}
+
+  it 'should let customers see their order' do
+    expect(subject.view_order).to eq ({})
```

Comment:
If you use parenthesis (which are optional), don't put a space between them and the statement